server/modules/analysis/repository/processingChainRepository.js

```diff
@@ -119,3 +119,12 @@ export const deleteChainsWithoutCycles = async (surveyId, client = db) =>
     DELETE FROM ${getSurveyDBSchema(surveyId)}.processing_chain
     WHERE jsonb_array_length(props->'${ProcessingChain.keysProps.cycles}') = 0
   `)
+
+// ===== GRANT PRIVILEGES
+export const grantUpdateOnProcessingChainStatusToUser = async (surveyId, userName, client = db) =>
```

Comment:
should we shorten the name? e.g. grantUpdateToAnalysisUser?

Reply:
ok